src/net/sourceforge/kolmafia/swingui/CalendarFrame.java

```diff
@@ -85,7 +84,7 @@ private void buildCalendarFrame(Calendar calendarToUse) {
     CalendarFrame.selectedDate = calendarToUse;
 
     CalendarFrame.calculatePhases(
-        CalendarFrame.selectedDate.toInstant().atZone(ZoneId.systemDefault()));
+        CalendarFrame.selectedDate.toInstant().atZone(KoLmafia.KOL_TIME_ZONE.toZoneId()));
```

Comment:
I am somewhat vindicated because the the .systemdefault() was a change from 2022 and not mine.  I am, however, distressed that my test was inadequate.

Reply:
Your test was fine for most of the day :)

It was only between rollover and midnight (server local time) that it ran into issues.